node.go

```diff
@@ -832,15 +832,19 @@ type HistoryResult struct {
 	Publications []*protocol.Publication
 }
 
-// History returns a slice of last messages published into project channel.
-func (n *Node) History(ch string) (HistoryResult, error) {
-	actionCount.WithLabelValues("history").Inc()
+// History allows to extract Publications in channel.
+// The channel must belong to namespace where history is on.
+func (n *Node) History(ch string, opts ...HistoryOption) (HistoryResult, error) {
+	historyOpts := &HistoryOptions{}
+	for _, opt := range opts {
+		opt(historyOpts)
+	}
 	if n.historyManager == nil {
```

Comment:
Probably this is can be moved before option loop, unless we can modify `n. historyManager`

Reply:
good point, just moved it to top, also found that some similar checks can be removed from helper functions.